From e8ebb586cfe9e2374d6a68345169351e9ac3604f Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Sun, 13 Mar 2022 14:21:49 -0600
Subject: [PATCH] Don't use the full paths in the .pc

---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 7705432..02ef91e 100644
--- a/Makefile
+++ b/Makefile
@@ -428,7 +428,7 @@ regress: bins
 	convert $< -thumbnail 350 -quality 50 $@
 
 .in.pc.pc:
-	sed -e "s!@PREFIX@!$(PREFIX)!g" \
-	    -e "s!@LIBDIR@!$(LIBDIR)!g" \
-	    -e "s!@INCLUDEDIR@!$(INCLUDEDIR)!g" \
+	sed -e "s!@PREFIX@!$(prefix)!g" \
+	    -e "s!@LIBDIR@!$(libdir)!g" \
+	    -e "s!@INCLUDEDIR@!$(includedir)!g" \
 	    -e "s!@VERSION@!$(VERSION)!g" $< >$@
