From dedd6c12cbce3081ca91866f9c41c378264ae55c Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Sun, 13 Mar 2022 14:26:12 -0600
Subject: [PATCH] No documentation

---
 Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile b/Makefile
index e6ce50cbd..88b0d274d 100644
--- a/Makefile
+++ b/Makefile
@@ -19,7 +19,6 @@ makefiles = \
   misc/systemd/local.mk \
   misc/launchd/local.mk \
   misc/upstart/local.mk \
-  doc/manual/local.mk \
   tests/local.mk \
   tests/plugins/local.mk
 
