From de0a0097a0fa43219ae2183c06a45ee22a3c9b0d Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Sun, 13 Mar 2022 14:31:13 -0600
Subject: [PATCH] Force cast

---
 src/libexpr/eval.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/libexpr/eval.cc b/src/libexpr/eval.cc
index b884b4001..c1374c166 100644
--- a/src/libexpr/eval.cc
+++ b/src/libexpr/eval.cc
@@ -42,7 +42,7 @@ static char * allocString(size_t size)
 #if HAVE_BOEHMGC
     t = (char *) GC_MALLOC_ATOMIC(size);
 #else
-    t = malloc(size);
+    t = (char *) malloc(size);
 #endif
     if (!t) throw std::bad_alloc();
     return t;
