From f9b27c002619dc0790a629f81d658f42e2aa3535 Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Tue, 22 Mar 2022 15:59:18 -0600
Subject: [PATCH] no allocators if don't HAVE_BOEHMGC

---
 src/libexpr/eval.cc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/libexpr/eval.cc b/src/libexpr/eval.cc
index 0f08e6e90..87c6b086a 100644
--- a/src/libexpr/eval.cc
+++ b/src/libexpr/eval.cc
@@ -456,9 +456,6 @@ EvalState::EvalState(
 #if HAVE_BOEHMGC
     , valueAllocCache(std::allocate_shared<void *>(traceable_allocator<void *>(), nullptr))
     , env1AllocCache(std::allocate_shared<void *>(traceable_allocator<void *>(), nullptr))
-#else
-    , valueAllocCache(std::make_shared<void *>(nullptr))
-    , env1AllocCache(std::make_shared<void *>(nullptr))
 #endif
     , baseEnv(allocEnv(128))
     , staticBaseEnv(false, 0)
