From ad6541e99ed9705c6989065dfb22f6b90f1c3428 Mon Sep 17 00:00:00 2001
From: Nigel Whillier <nigel.w@nosun.ca>
Date: Wed, 23 Mar 2022 09:19:30 -0600
Subject: [PATCH] Don't split nix into shared libs

---
 mk/lib.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mk/lib.mk b/mk/lib.mk
index 92f0983d5..7cb15b365 100644
--- a/mk/lib.mk
+++ b/mk/lib.mk
@@ -64,7 +64,7 @@ endif
 
 
 # Pass -fPIC if we're building dynamic libraries.
-BUILD_SHARED_LIBS ?= 1
+BUILD_SHARED_LIBS = 0
 
 ifeq ($(BUILD_SHARED_LIBS), 1)
   ifdef HOST_CYGWIN
